Update docs example for multiple app stanza (#929)
hashicorp/waypoint: website/content/docs/waypoint-hcl/app.mdx

A minimum of one `app` stanza is required in your project configuration.

```hcl
app "frontend" {
  build {
    # ...
  }

  deploy {
    # ...
  }

  release {
    # ...
  }
}
```

Multiple `app` stanzas can be specified if Waypoint should deploy multiple

> applications in your project. An example where this might be useful is

Reviewer: ```suggestion
applications in your project. Some examples where this might be useful are
```

> separate backend and frontend processes or your project is monorepo.

Reviewer 2: ```suggestion
separate backend and frontend processes or with a project that is a monorepo.
```

> separate backend and frontend processes or with a project that is a monorepo.

Reviewer: ```suggestion
a deployment that targets multiple application tiers, a project that is part of a monorepo, or applications that require lock-step deployment.
```

```hcl

> project = "monorepo-project"

Reviewer: ```suggestion
project = "multi-tier-project"
```

    # ...
  }
}

app "backend" {
  build {
    # ...
  }

  deploy {
    # ...
  }

  release {
    # ...
  }
}
```

> And then we can select specified `app` with paramater `-app <app-label>` . 
> An example we want deploy frontend `app` on monorepo project, we can run 
> `waypoint deploy -app frontend` for deploying frontend `app`.
> Also can be used in `waypoint build` , `deploy` , `release` and `up`

Reviewer 2: ```suggestion
Select a specified `app` with the paramater `-app <app-label>`. For example, 
to deploy the `frontend` app from the `monorepo-project`, run 
`waypoint deploy -app frontend`.
The `-app` flag is a global target, and can be used with all Waypoint 
commands.
```

    # ...
  }
}

app "backend" {
  build {
    # ...
  }

  deploy {
    # ...
  }

  release {
    # ...
  }
}
```

Select a specified `app` with the paramater `-app <app-label>`. For example, 

> to deploy the `frontend` app from the `monorepo-project`, run 

Reviewer: ```suggestion
to deploy the `frontend` app from the `multi-tier-project`, run 
```
